tests/integration/test_webhook.py

```diff
@@ -136,3 +136,29 @@ def test_integration_webhook_handles_timdex_step_function_trigger(
     assert step_function_input_json["run-date"] == "2023-08-15"
     assert step_function_input_json["run-type"] == "daily"
     assert step_function_input_json["source"] == "alma"
+
+
+@pytest.mark.integration()
+@pytest.mark.usefixtures("_set_integration_test_environ")
+@pytest.mark.usefixtures("_integration_tests_s3_fixtures")
+def test_integration_webhook_handles_bursar_step_function_trigger(
+    sample_bursar_export_job_end_webhook_post_body,
+):
+    """Test deployed lambda handles PPOD job type webhooks and invokes step function.
```

Comment:
The perpetual curse of docstrings (really not always required in tests, but sometimes I think complex tests warrant them).  Think this should reference "Bursar" instead of "PPOD"?